project_hr/tests/test_project_hr.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import SavepointCase
+from odoo.exceptions import ValidationError
+
+
+class TestProjectHr(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.user = cls.env['res.users'].create({
```

Comment:
Here `cls.env['res.users']`, `cls.env['hr.employee.category']`  use multiple time, use as a global variable

Reply:
Although not critical, and most as it only appears 2 times, it's done!